pkg/resource/deploy/deploytest/pluginhost.go

```diff
@@ -91,8 +93,12 @@ func (host *pluginHost) ServerAddr() string {
 	panic("Host RPC address not available")
 }
 func (host *pluginHost) Log(sev diag.Severity, urn resource.URN, msg string, streamID int32,
-	isStatus bool) {
-	host.sink.Logf(sev, diag.StreamMessage(urn, msg, streamID), isStatus)
+	ephemeral bool) {
+	if ephemeral {
+		host.statusSink.Logf(sev, diag.StreamMessage(urn, msg, streamID))
+	} else {
+		host.sink.Logf(sev, diag.StreamMessage(urn, msg, streamID))
+	}
 }
```

Comment:
Any reason not to add a `LogStatus` to this API that does the `statusSink` version?  That avoids forcing `, false` into a ton of places inside providers.

Reply:
I don't understand this suggestion. You'd still have to add `, &LogStatus{}`, no?

Reply:
I mean two methods:

```go
host.Log()
host.LogStatus()
```

Reply:
I'll admit, I wondered the same.